cfme/tests/infrastructure/test_quota.py

```diff
@@ -98,6 +98,78 @@ def entities(appliance, request, max_quota_test_instance):
     return getattr(appliance.collections, collection).instantiate(description)
 
 
+@pytest.fixture(scope='module')
+def new_tenant(appliance):
+    tenant_list = []
+    for i in range(0, 3):
```

Comment:
Move this upper limit that's used across the fixtures into a module constant, `NUM_GROUPS`, and use it so they stay consistent and we potentially avoid any IndexError if one of them is changed.

Reply:
Yep, and then re-use NUM_GROUPS - if you want separate names, you can do something like:
```
NUM_GROUPS = NUM_TENANTS = NUM_WHATEVER = 3